From 550c1389e4ffb6e5c184a574f7eca49ad4314223 Mon Sep 17 00:00:00 2001
From: James Kalb <James.Kalbfleisch@lairdtech.com>
Date: Wed, 31 Aug 2016 16:05:09 -0400
Subject: [PATCH] laird:  Enabling the BT config.

The ss1btusb driver is failing to build because of the dependency of
CONFIG_BT which was removed at some point after GA5.  Restoring this
config will re-enable the driver build and the SS1 stack to be
functional again.

Bug 9750
---
 arch/arm/configs/wb45n_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/wb45n_defconfig b/arch/arm/configs/wb45n_defconfig
index cb27ea7726d3..44e0a6983251 100644
--- a/arch/arm/configs/wb45n_defconfig
+++ b/arch/arm/configs/wb45n_defconfig
@@ -116,6 +116,7 @@ CONFIG_BRIDGE_EBT_LOG=m
 CONFIG_BRIDGE_EBT_NFLOG=m
 CONFIG_BRIDGE=m
 CONFIG_NET_PKTGEN=m
+CONFIG_BT=m
 CONFIG_BT_HCISS1USB=m
 CONFIG_CFG80211=m
 CONFIG_NL80211_TESTMODE=y
-- 
2.11.0

